docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
     environment:
       - HTFEED_CONFIG=/usr/local/feed/etc/config_test.yml
       - FEED_HOME=/usr/local/feed
-    command: make test TEST_VERBOSE=1
+    command: bin/wait-for --timeout=300 mariadb:3306 -- make test TEST_VERBOSE=1
```

Comment:
I can't explain why, but I love this.

Reply:
Cribbed from the "docker for rails" book, which recommends using this approach for a one-shot db:migrate container